middle-pgsql.cpp

```diff
@@ -313,7 +313,7 @@ int pgsql_endCopy(middle_pgsql_t::table_desc *table)
 }
 } // anonymous namespace
 
-int middle_pgsql_t::local_nodes_set(const osmid_t& id, const double& lat,
+bool middle_pgsql_t::local_nodes_set(const osmid_t& id, const double& lat,
```

Comment:
These changes to bool look wrong. The functions used to return 0 for ok, which now translates to false. Either also change the return statements properly or drop the return value completely if it isn't used anywhere.

This concerns all bool returns in this PR.